### PR TITLE
Create GNSS documents bucket using terraform

### PR DESCRIPTION
**Pull request by frankfu03**
The name of the S3 bucket ( gnss-documents-{env} ) is open for discussion. There is a bucked named ga-gnss-documentation in s3, which has no environment suffix.

### CODE REVIEW

File: aws/terraform/s3bucket.tf [hunk 1]
```diff
@@ -0,0 +1,15 @@
+resource "aws_s3_bucket" "gnss-documents" {
```
Comment (lbodor) [inline aws/terraform/s3bucket.tf]: underscores here: gnss_documents

Comment (frankfu03) [reply aws/terraform/s3bucket.tf]: Ok, it is done.

**Comment by frankfu03** [comment]
As suggested, I have rename the S3 bucket to "gnss-metadata-document-storage-${var.environment}".

**Comment by frankfu03** [comment]
The PR is ready for review again.

**Comment by frankfu03** [comment]
As suggested, I have rename s3bucket.tf to document_storage.tf.

**Pull request merged**